x/distribution/keeper/msg_server.go

```diff
@@ -101,21 +104,40 @@ func (k msgServer) WithdrawValidatorCommission(ctx context.Context, msg *types.M
 	return &types.MsgWithdrawValidatorCommissionResponse{Amount: amount}, nil
 }
 
-func (k msgServer) FundCommunityPool(ctx context.Context, msg *types.MsgFundCommunityPool) (*types.MsgFundCommunityPoolResponse, error) {
-	depositor, err := k.authKeeper.AddressCodec().StringToBytes(msg.Depositor)
-	if err != nil {
-		return nil, sdkerrors.ErrInvalidAddress.Wrapf("invalid depositor address: %s", err)
-	}
+// NOTE: DO NOT USE
+// This method uses deprecated message request. Use FundCommunityPool from x/protocolpool module instead.
+func (k msgServer) FundCommunityPool(ctx context.Context, msg *types.MsgFundCommunityPool) (*types.MsgFundCommunityPoolResponse, error) { //nolint:staticcheck // we're using a deprecated call for compatibility
```

Comment:
Tests have been removed for this, can we get a test to verify the fallback works?
Same for queries.